segmentation/include/pcl/segmentation/impl/region_growing_rgb.hpp

```diff
@@ -338,7 +338,7 @@ pcl::RegionGrowingRGB<PointT, NormalT>::findRegionsKNN (pcl::index_t index, pcl:
   {
     if (distances[i_seg] < max_dist)
     {
-      segment_neighbours.push (std::make_pair (distances[i_seg], i_seg) );
+      segment_neighbours.emplace (std::make_pair (distances[i_seg], i_seg) );
```

Comment:
```suggestion
      segment_neighbours.emplace (distances[i_seg], i_seg);
```
This is what clang-tidy suggests: https://github.com/PointCloudLibrary/pcl/actions/runs/10539455724/job/29246218780#step:4:2805
```
/__w/pcl/pcl/segmentation/include/pcl/segmentation/impl/region_growing_rgb.hpp:341:26: error: use emplace instead of push [modernize-use-emplace,-warnings-as-errors]
  341 |       segment_neighbours.push (std::make_pair (distances[i_seg], i_seg) );
      |                          ^~~~~~~~~~~~~~~~~~~~                         ~
      |                          emplace
```

Reply:
Right. My bad.